prefs.py

```diff
@@ -251,30 +223,21 @@ def __init__(self, parent: tk.Tk, callback: Optional[Callable]):
 
         self.parent = parent
         self.callback = callback
-        if sys.platform == 'darwin':
-            # LANG: File > Preferences menu entry for macOS
-            self.title(_('Preferences'))
-
-        else:
-            # LANG: File > Settings (macOS)
-            self.title(_('Settings'))
+        # LANG: File > Settings (macOS)
+        self.title(_('Settings'))
 
         if parent.winfo_viewable():
             self.transient(parent)
 
         # position over parent
-        if sys.platform != 'darwin' or parent.winfo_rooty() > 0:  # http://core.tcl.tk/tk/tktview/c84f660833546b1b84e7
+        if parent.winfo_rooty() > 0:  # http://core.tcl.tk/tk/tktview/c84f660833546b1b84e7
```

Comment:
Warning, logic change: ``parent.winfo_rooty() > 0`` only got checked for ``macOS``.